packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -268,11 +269,24 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin
 
     _position = controller?.createScrollPosition(_physics, this, oldPosition)
       ?? new ScrollPositionWithSingleContext(physics: _physics, context: this, oldPosition: oldPosition);
+    _position.addListener(_sendSemanticsScrollEvent);
 
     assert(position != null);
     controller?.attach(position);
   }
 
+  bool _semanticsScrollEventScheduled = false;
+
+  void _sendSemanticsScrollEvent() {
+    if (_semanticsScrollEventScheduled)
+      return;
+    SchedulerBinding.instance.addPostFrameCallback((Duration timestamp) {
+      _gestureDetectorKey.currentState?.sendSemanticsEvent(new ScrollSemanticsEvent());
```

Comment:
should you check if everything is still mounted at this point?

Reply:
`SemanticsNode.sendEvent` checks if the SemanticsNode is still attached and only sends the event if it is.